src/main/java/org/opentripplanner/updater/UpdatersParameters.java

```diff
@@ -17,7 +18,7 @@
 
 public interface UpdatersParameters {
 
-  URI bikeRentalServiceDirectoryUrl();
+  BikeRentalServiceDirectoryFetcherParameters getBikeRentalServiceDirectoryFetcherParameters();
```

Comment:
This will cause circular dependencies between the Sandbox code and the main code. You should move the `BikeRentalServiceDirectoryFetcherParameters` to a sub-package, like `o.o.ext.bikerentalservicedirectory.api` or 
`o.o.ext.bikerentalservicedirectory.parameters`.